docs/source/getting_started/command_line_tool.rst

```diff
@@ -286,6 +214,32 @@ Again, you can use the *dataset_id* and *new_dataset_name* interchangeably.
                    token=your_token new_dataset_name=your_dataset_name
 
 
+.. _ref-upload-custom-metadata-lightly:
+
+Upload custom metadata using the CLI
+---------------------------------------
+    
+You can upload custom metadata along with your images. Custom metadata can be used
+to gain additional insights in the web-app (see :ref:`platform-custom-metadata`).
```

Comment:
I think we call it `lightly platform` rather than web-app everywhere else. I think we should be consistent or call it `lightly platform/web-app`